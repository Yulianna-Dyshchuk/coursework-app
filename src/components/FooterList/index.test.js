import React from "react";
import { shallow } from "enzyme";
import { FooterList } from "./index";

describe("FooterList component", () => {
  it("should render correctly without props", () => {
    const component = shallow(<FooterList />);

    expect(component).toMatchSnapshot();
  });

  it("should render correctly with props", () => {
    const component = shallow(<FooterList title="test" list={[]} />);

    expect(component).toMatchSnapshot();
  });

  it("should render correctly with props and children", () => {
    const component = shallow(
      <FooterList title="test" list={[]}>
        <span>test child 1</span>
        <span>test child 2</span>
      </FooterList>
    );

    expect(component).toMatchSnapshot();
  });
});
