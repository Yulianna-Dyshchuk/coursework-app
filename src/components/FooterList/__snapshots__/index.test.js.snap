// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterList component should render correctly with props 1`] = `
<div>
  <WithStyles(ForwardRef(Typography))
    variant="subtitle1"
  >
    test
  </WithStyles(ForwardRef(Typography))>
</div>
`;

exports[`FooterList component should render correctly with props and children 1`] = `
<div>
  <WithStyles(ForwardRef(Typography))
    variant="subtitle1"
  >
    test
  </WithStyles(ForwardRef(Typography))>
  <span>
    test child 1
  </span>
  <span>
    test child 2
  </span>
</div>
`;

exports[`FooterList component should render correctly without props 1`] = `
<div>
  <WithStyles(ForwardRef(Typography))
    variant="subtitle1"
  />
</div>
`;
